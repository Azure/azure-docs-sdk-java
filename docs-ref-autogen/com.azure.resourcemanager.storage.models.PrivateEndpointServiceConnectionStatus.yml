### YamlMime:JavaType
uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus"
fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus"
name: "PrivateEndpointServiceConnectionStatus"
nameWithType: "PrivateEndpointServiceConnectionStatus"
summary: "The private endpoint connection status."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedClassMethods:
- classRef: "<xref href=\"com.azure.core.util.ExpandableStringEnum?alt=com.azure.core.util.ExpandableStringEnum&text=ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)?alt=com.azure.core.util.ExpandableStringEnum.<T>fromString&text=<T>fromString\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)?alt=com.azure.core.util.ExpandableStringEnum.<T>values&text=<T>values\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)?alt=com.azure.core.util.ExpandableStringEnum.equals&text=equals\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.hashCode()?alt=com.azure.core.util.ExpandableStringEnum.hashCode&text=hashCode\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.toString()?alt=com.azure.core.util.ExpandableStringEnum.toString&text=toString\" data-throw-if-not-resolved=\"False\" />"
- classRef: "java.lang.<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html\">Object</a>"
  methodsRef:
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#clone--\">clone</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#finalize--\">finalize</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#getClass--\">getClass</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notify--\">notify</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notifyAll--\">notifyAll</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait--\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-int-\">wait</a>"
syntax: "public final class PrivateEndpointServiceConnectionStatus extends ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>"
constructors:
- uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  name: "PrivateEndpointServiceConnectionStatus()"
  nameWithType: "PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  syntax: "public PrivateEndpointServiceConnectionStatus()"
fields:
- uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
  desc: "Static value Approved for PrivateEndpointServiceConnectionStatus."
- uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.PENDING"
  fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.PENDING"
  name: "PENDING"
  nameWithType: "PrivateEndpointServiceConnectionStatus.PENDING"
  summary: "Static value Pending for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus PENDING"
  desc: "Static value Pending for PrivateEndpointServiceConnectionStatus."
- uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.REJECTED"
  fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.REJECTED"
  name: "REJECTED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.REJECTED"
  summary: "Static value Rejected for Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus REJECTED"
  desc: "Static value Rejected for PrivateEndpointServiceConnectionStatus."
methods:
- uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)"
  fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "PrivateEndpointServiceConnectionStatus.fromString(String name)"
  summary: "Creates or finds a Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for."
    name: "name"
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/String.html\">String</a>"
  syntax: "public static PrivateEndpointServiceConnectionStatus fromString(String name)"
  desc: "Creates or finds a PrivateEndpointServiceConnectionStatus from its string representation."
  returns:
    description: "the corresponding PrivateEndpointServiceConnectionStatus."
    type: "<xref href=\"com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.values()"
  fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.values()"
  name: "values()"
  nameWithType: "PrivateEndpointServiceConnectionStatus.values()"
  summary: "Gets known Private<wbr>Endpoint<wbr>Service<wbr>Connection<wbr>Status values."
  modifiers:
  - "static"
  syntax: "public static Collection<PrivateEndpointServiceConnectionStatus> values()"
  desc: "Gets known PrivateEndpointServiceConnectionStatus values."
  returns:
    description: "known PrivateEndpointServiceConnectionStatus values."
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html\">Collection</a>&lt;<xref href=\"com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
desc: "The private endpoint connection status."
metadata: {}
package: "com.azure.resourcemanager.storage.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-storage:2.19.0
