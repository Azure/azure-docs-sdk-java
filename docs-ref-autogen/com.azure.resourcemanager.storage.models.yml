### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.storage.models"
fullName: "com.azure.resourcemanager.storage.models"
name: "com.azure.resourcemanager.storage.models"
summary: "Package containing the data models for StorageManagementClient. The Azure Storage Management API."
classes:
- "com.azure.resourcemanager.storage.models.AccountSasParameters"
- "com.azure.resourcemanager.storage.models.AccountStatuses"
- "com.azure.resourcemanager.storage.models.ActiveDirectoryProperties"
- "com.azure.resourcemanager.storage.models.AzureEntityResource"
- "com.azure.resourcemanager.storage.models.AzureFilesIdentityBasedAuthentication"
- "com.azure.resourcemanager.storage.models.BlobContainersCreateOrUpdateImmutabilityPolicyHeaders"
- "com.azure.resourcemanager.storage.models.BlobContainersCreateOrUpdateImmutabilityPolicyResponse"
- "com.azure.resourcemanager.storage.models.BlobContainersDeleteImmutabilityPolicyHeaders"
- "com.azure.resourcemanager.storage.models.BlobContainersDeleteImmutabilityPolicyResponse"
- "com.azure.resourcemanager.storage.models.BlobContainersExtendImmutabilityPolicyHeaders"
- "com.azure.resourcemanager.storage.models.BlobContainersExtendImmutabilityPolicyResponse"
- "com.azure.resourcemanager.storage.models.BlobContainersGetImmutabilityPolicyHeaders"
- "com.azure.resourcemanager.storage.models.BlobContainersGetImmutabilityPolicyResponse"
- "com.azure.resourcemanager.storage.models.BlobContainersLockImmutabilityPolicyHeaders"
- "com.azure.resourcemanager.storage.models.BlobContainersLockImmutabilityPolicyResponse"
- "com.azure.resourcemanager.storage.models.BlobRestoreParameters"
- "com.azure.resourcemanager.storage.models.BlobRestoreProgressStatus"
- "com.azure.resourcemanager.storage.models.BlobRestoreRange"
- "com.azure.resourcemanager.storage.models.BlobServiceItems"
- "com.azure.resourcemanager.storage.models.BlobTypes"
- "com.azure.resourcemanager.storage.models.Bypass"
- "com.azure.resourcemanager.storage.models.ChangeFeed"
- "com.azure.resourcemanager.storage.models.CheckNameAvailabilityResult"
- "com.azure.resourcemanager.storage.models.CorsRule"
- "com.azure.resourcemanager.storage.models.CorsRuleAllowedMethodsItem"
- "com.azure.resourcemanager.storage.models.CorsRules"
- "com.azure.resourcemanager.storage.models.CustomDomain"
- "com.azure.resourcemanager.storage.models.DateAfterCreation"
- "com.azure.resourcemanager.storage.models.DateAfterModification"
- "com.azure.resourcemanager.storage.models.DeleteRetentionPolicy"
- "com.azure.resourcemanager.storage.models.DeletedShare"
- "com.azure.resourcemanager.storage.models.Dimension"
- "com.azure.resourcemanager.storage.models.DirectoryServiceOptions"
- "com.azure.resourcemanager.storage.models.EnabledProtocols"
- "com.azure.resourcemanager.storage.models.Encryption"
- "com.azure.resourcemanager.storage.models.EncryptionScopeKeyVaultProperties"
- "com.azure.resourcemanager.storage.models.EncryptionScopeListResult"
- "com.azure.resourcemanager.storage.models.EncryptionScopeSource"
- "com.azure.resourcemanager.storage.models.EncryptionScopeState"
- "com.azure.resourcemanager.storage.models.EncryptionService"
- "com.azure.resourcemanager.storage.models.EncryptionServices"
- "com.azure.resourcemanager.storage.models.Endpoints"
- "com.azure.resourcemanager.storage.models.FileShareItems"
- "com.azure.resourcemanager.storage.models.GeoReplicationStats"
- "com.azure.resourcemanager.storage.models.GeoReplicationStatus"
- "com.azure.resourcemanager.storage.models.Identity"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicyProperties"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicyState"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicyUpdateType"
- "com.azure.resourcemanager.storage.models.IpRule"
- "com.azure.resourcemanager.storage.models.KeySource"
- "com.azure.resourcemanager.storage.models.KeyType"
- "com.azure.resourcemanager.storage.models.KeyVaultProperties"
- "com.azure.resourcemanager.storage.models.Kind"
- "com.azure.resourcemanager.storage.models.LargeFileSharesState"
- "com.azure.resourcemanager.storage.models.LeaseContainerRequest"
- "com.azure.resourcemanager.storage.models.LeaseContainerRequestAction"
- "com.azure.resourcemanager.storage.models.LeaseDuration"
- "com.azure.resourcemanager.storage.models.LeaseState"
- "com.azure.resourcemanager.storage.models.LeaseStatus"
- "com.azure.resourcemanager.storage.models.LegalHoldProperties"
- "com.azure.resourcemanager.storage.models.ListContainerItems"
- "com.azure.resourcemanager.storage.models.ListContainersInclude"
- "com.azure.resourcemanager.storage.models.ListQueueResource"
- "com.azure.resourcemanager.storage.models.ListTableResource"
- "com.azure.resourcemanager.storage.models.ManagementPolicyAction"
- "com.azure.resourcemanager.storage.models.ManagementPolicyBaseBlob"
- "com.azure.resourcemanager.storage.models.ManagementPolicyDefinition"
- "com.azure.resourcemanager.storage.models.ManagementPolicyFilter"
- "com.azure.resourcemanager.storage.models.ManagementPolicyName"
- "com.azure.resourcemanager.storage.models.ManagementPolicyRule"
- "com.azure.resourcemanager.storage.models.ManagementPolicySchema"
- "com.azure.resourcemanager.storage.models.ManagementPolicySnapShot"
- "com.azure.resourcemanager.storage.models.MetricSpecification"
- "com.azure.resourcemanager.storage.models.MinimumTlsVersion"
- "com.azure.resourcemanager.storage.models.NetworkRuleSet"
- "com.azure.resourcemanager.storage.models.ObjectReplicationPolicies"
- "com.azure.resourcemanager.storage.models.ObjectReplicationPolicyFilter"
- "com.azure.resourcemanager.storage.models.ObjectReplicationPolicyRule"
- "com.azure.resourcemanager.storage.models.OperationDisplay"
- "com.azure.resourcemanager.storage.models.OperationListResult"
- "com.azure.resourcemanager.storage.models.Permissions"
- "com.azure.resourcemanager.storage.models.PrivateEndpoint"
- "com.azure.resourcemanager.storage.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.storage.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.storage.models.PrivateLinkResource"
- "com.azure.resourcemanager.storage.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.storage.models.PublicEndpoints"
- "com.azure.resourcemanager.storage.models.ReasonCode"
- "com.azure.resourcemanager.storage.models.RestorePolicyProperties"
- "com.azure.resourcemanager.storage.models.Restriction"
- "com.azure.resourcemanager.storage.models.RootSquashType"
- "com.azure.resourcemanager.storage.models.RoutingChoice"
- "com.azure.resourcemanager.storage.models.RoutingPreference"
- "com.azure.resourcemanager.storage.models.RuleType"
- "com.azure.resourcemanager.storage.models.ServiceSasParameters"
- "com.azure.resourcemanager.storage.models.ServiceSpecification"
- "com.azure.resourcemanager.storage.models.Services"
- "com.azure.resourcemanager.storage.models.ShareAccessTier"
- "com.azure.resourcemanager.storage.models.SignedResource"
- "com.azure.resourcemanager.storage.models.SignedResourceTypes"
- "com.azure.resourcemanager.storage.models.Sku"
- "com.azure.resourcemanager.storage.models.SkuCapability"
- "com.azure.resourcemanager.storage.models.SkuName"
- "com.azure.resourcemanager.storage.models.StorageAccountCheckNameAvailabilityParameters"
- "com.azure.resourcemanager.storage.models.StorageAccountCreateParameters"
- "com.azure.resourcemanager.storage.models.StorageAccountEncryptionKeySource"
- "com.azure.resourcemanager.storage.models.StorageAccountInternetEndpoints"
- "com.azure.resourcemanager.storage.models.StorageAccountKey"
- "com.azure.resourcemanager.storage.models.StorageAccountListResult"
- "com.azure.resourcemanager.storage.models.StorageAccountMicrosoftEndpoints"
- "com.azure.resourcemanager.storage.models.StorageAccountRegenerateKeyParameters"
- "com.azure.resourcemanager.storage.models.StorageAccountSkuType"
- "com.azure.resourcemanager.storage.models.StorageAccountUpdateParameters"
- "com.azure.resourcemanager.storage.models.StorageResourceType"
- "com.azure.resourcemanager.storage.models.StorageService"
- "com.azure.resourcemanager.storage.models.StorageSkuListResult"
- "com.azure.resourcemanager.storage.models.TagFilter"
- "com.azure.resourcemanager.storage.models.TagProperty"
- "com.azure.resourcemanager.storage.models.UpdateHistoryProperty"
- "com.azure.resourcemanager.storage.models.UsageListResult"
- "com.azure.resourcemanager.storage.models.UsageName"
- "com.azure.resourcemanager.storage.models.VirtualNetworkRule"
enums:
- "com.azure.resourcemanager.storage.models.AccessTier"
- "com.azure.resourcemanager.storage.models.AccountStatus"
- "com.azure.resourcemanager.storage.models.Action"
- "com.azure.resourcemanager.storage.models.DefaultAction"
- "com.azure.resourcemanager.storage.models.GetShareExpand"
- "com.azure.resourcemanager.storage.models.HttpProtocol"
- "com.azure.resourcemanager.storage.models.KeyPermission"
- "com.azure.resourcemanager.storage.models.ListKeyExpand"
- "com.azure.resourcemanager.storage.models.ListSharesExpand"
- "com.azure.resourcemanager.storage.models.ProvisioningState"
- "com.azure.resourcemanager.storage.models.PublicAccess"
- "com.azure.resourcemanager.storage.models.Reason"
- "com.azure.resourcemanager.storage.models.SkuTier"
- "com.azure.resourcemanager.storage.models.State"
- "com.azure.resourcemanager.storage.models.StorageAccountExpand"
- "com.azure.resourcemanager.storage.models.UsageUnit"
interfaces:
- "com.azure.resourcemanager.storage.models.BlobContainer"
- "com.azure.resourcemanager.storage.models.BlobContainer.Definition"
- "com.azure.resourcemanager.storage.models.BlobContainer.DefinitionStages"
- "com.azure.resourcemanager.storage.models.BlobContainer.DefinitionStages.Blank"
- "com.azure.resourcemanager.storage.models.BlobContainer.DefinitionStages.WithBlobService"
- "com.azure.resourcemanager.storage.models.BlobContainer.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.storage.models.BlobContainer.DefinitionStages.WithMetadata"
- "com.azure.resourcemanager.storage.models.BlobContainer.DefinitionStages.WithPublicAccess"
- "com.azure.resourcemanager.storage.models.BlobContainer.Update"
- "com.azure.resourcemanager.storage.models.BlobContainer.UpdateStages"
- "com.azure.resourcemanager.storage.models.BlobContainer.UpdateStages.WithMetadata"
- "com.azure.resourcemanager.storage.models.BlobContainer.UpdateStages.WithPublicAccess"
- "com.azure.resourcemanager.storage.models.BlobContainers"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.Definition"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.DefinitionStages"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.DefinitionStages.Blank"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.DefinitionStages.WithCors"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.DefinitionStages.WithDefaultServiceVersion"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.DefinitionStages.WithDeleteRetentionPolicy"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.DefinitionStages.WithStorageAccount"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.Update"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.UpdateStages"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.UpdateStages.WithCors"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.UpdateStages.WithDefaultServiceVersion"
- "com.azure.resourcemanager.storage.models.BlobServiceProperties.UpdateStages.WithDeleteRetentionPolicy"
- "com.azure.resourcemanager.storage.models.BlobServices"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.Definition"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.DefinitionStages"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.DefinitionStages.Blank"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.DefinitionStages.WithContainer"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.DefinitionStages.WithETagCheck"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.DefinitionStages.WithImmutabilityPeriodSinceCreationInDays"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.Update"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.UpdateStages"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.UpdateStages.WithETagCheck"
- "com.azure.resourcemanager.storage.models.ImmutabilityPolicy.UpdateStages.WithImmutabilityPeriodSinceCreationInDays"
- "com.azure.resourcemanager.storage.models.LegalHold"
- "com.azure.resourcemanager.storage.models.ManagementPolicies"
- "com.azure.resourcemanager.storage.models.ManagementPolicy"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.Definition"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.DefinitionStages"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.DefinitionStages.Blank"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.DefinitionStages.WithRule"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.DefinitionStages.WithStorageAccount"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.Update"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.UpdateStages"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.UpdateStages.Rule"
- "com.azure.resourcemanager.storage.models.ManagementPolicy.UpdateStages.WithPolicy"
- "com.azure.resourcemanager.storage.models.PolicyRule"
- "com.azure.resourcemanager.storage.models.PolicyRule.Definition"
- "com.azure.resourcemanager.storage.models.PolicyRule.DefinitionStages"
- "com.azure.resourcemanager.storage.models.PolicyRule.DefinitionStages.Blank"
- "com.azure.resourcemanager.storage.models.PolicyRule.DefinitionStages.PrefixActionFork"
- "com.azure.resourcemanager.storage.models.PolicyRule.DefinitionStages.WithBlobTypesToFilterFor"
- "com.azure.resourcemanager.storage.models.PolicyRule.DefinitionStages.WithPolicyRuleAttachable"
- "com.azure.resourcemanager.storage.models.PolicyRule.DefinitionStages.WithPolicyRuleType"
- "com.azure.resourcemanager.storage.models.PolicyRule.DefinitionStages.WithPrefixesToFilterFor"
- "com.azure.resourcemanager.storage.models.PolicyRule.DefinitionStages.WithRuleActions"
- "com.azure.resourcemanager.storage.models.PolicyRule.Update"
- "com.azure.resourcemanager.storage.models.PolicyRule.UpdateStages"
- "com.azure.resourcemanager.storage.models.PolicyRule.UpdateStages.WithActions"
- "com.azure.resourcemanager.storage.models.PolicyRule.UpdateStages.WithBlobTypesToFilterFor"
- "com.azure.resourcemanager.storage.models.PolicyRule.UpdateStages.WithPrefixesToFilterFor"
- "com.azure.resourcemanager.storage.models.StorageAccount"
- "com.azure.resourcemanager.storage.models.StorageAccount.Definition"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.Blank"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithAccessTraffic"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithAzureFilesAadIntegration"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithBlobStorageAccountKind"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithBlockBlobStorageAccountKind"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithCreateAndAccessTier"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithCustomDomain"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithEncryption"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithFileStorageAccountKind"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithGeneralPurposeAccountKind"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithGroup"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithHns"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithLargeFileShares"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithManagedServiceIdentity"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithNetworkAccess"
- "com.azure.resourcemanager.storage.models.StorageAccount.DefinitionStages.WithSku"
- "com.azure.resourcemanager.storage.models.StorageAccount.Update"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages.WithAccessTier"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages.WithAccessTraffic"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages.WithCustomDomain"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages.WithEncryption"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages.WithManagedServiceIdentity"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages.WithNetworkAccess"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages.WithSku"
- "com.azure.resourcemanager.storage.models.StorageAccount.UpdateStages.WithUpgrade"
- "com.azure.resourcemanager.storage.models.StorageAccountEncryptionStatus"
- "com.azure.resourcemanager.storage.models.StorageAccounts"
- "com.azure.resourcemanager.storage.models.StorageSku"
- "com.azure.resourcemanager.storage.models.StorageSkus"
- "com.azure.resourcemanager.storage.models.Usages"
metadata: {}
package: "com.azure.resourcemanager.storage.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-storage:2.1.0
