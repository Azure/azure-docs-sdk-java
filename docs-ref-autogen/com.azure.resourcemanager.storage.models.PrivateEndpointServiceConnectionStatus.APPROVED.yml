### YamlMime:JavaMember
uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.storage.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.storage.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-storage:2.1.0
